docker/entrypoint.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+PROMPT_OPTS=()
+BOOTSTRAP_OPTS=()
+BOOTSTRAP=""
+MAIN_NET=""
+PRIV_NET=""
+COZ_NET=""
+
+echo "$1"
+
+while [ -n "$1" ]; do
+    case "$1" in
+        -m|--mainnet)
+            PROMPT_OPTS+="-m"
+            BOOTSTRAP_OPTS+="-m"
+            MAIN_NET="-m"
+        ;;
+        -p|--privnet)
+            PROMPT_OPTS+="-p"
+            PRIV_NET="-p"
+        ;;
+        --coznet)
+            PROMPT_OPTS+="--coznet"
```

Comment:
this should throw an error in the bootstrap step, else it downloads the testnet chain